motr/m0crate/btree_without_tx_seg.c

```diff
@@ -578,11 +480,6 @@ static void be_btree_insert_newkey(struct m0_be_btree      *btree,
 	struct m0_be_bnode *old_root;
 	struct m0_be_bnode *new_root;
 
-	M0_PRE(btree_invariant(btree));
```

Comment:
you should have kept the invariants if they are not related to segment or tx operations.

Reply:
Updated code to include invariants